build_tools/ci/cpu_comparison/run.py

```diff
@@ -78,9 +95,29 @@ def add_aie_compilation_flags(self, flags):
             self.aie_compilation_flags = list(set(self.aie_compilation_flags))
 
     def run(self, config):
-        if config.target_device in self.run_on_target:
-            return self._execute(config)
-        return False
+
+        # If the target device is not in the set of devices to run on, then
+        # return False. ie. don't raise an error because is legitimate,
+        # we just won't run the test.
+        if config.target_device not in self.run_on_target:
+            return False
+
+        # If use_chess=1, and config has not provided a valid
+        # path to vitis, then don't run the test. The asymettry between 
```

Comment:
Typo: asymettry -> asymmetry

Reply:
In future can you please make comments like this using the "Add suggestion" button? That way I can just land time directly in the web browser

![{5DFBAE28-FC09-4F07-ABE7-405C75D751D9}](https://github.com/user-attachments/assets/266dbd7f-e5d3-4e3f-81c0-c1ac0d47802c)